app/utils/bundle-importer.js

```diff
@@ -489,6 +493,21 @@ YUI.add('bundle-importer', function(Y) {
           }.bind(this),
           {modelId: relation.get('id')});
       next();
+    },
+
+    /**
+      Executes the setAnnotations method call
+
+      @method _execute_setAnnotations
+      @param {Object} record The setAnnotations record.
+      @param {Function} next The method to trigger the executor to move
+        on to the next record.
+    */
+    _execute_setAnnotations: function(record, next) {
+      var entityName = record[record.args[0].replace(/^\$/, '')].get('id');
+      var service = this.db.services.getById(entityName);
```

Comment:
Machines can have annotations, too, in the v4 format.  We may want to check that the requires/args[0] is an addService type.